modules/apps/foundation/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/search/bar/portlet/action/SearchBarRedirectMVCActionCommand.java

```diff
@@ -91,6 +98,14 @@ protected void doProcessAction(
 			searchBarPortletPreferences.getKeywordsParameterName(),
 			searchBarPortletPreferences.getScopeParameterName());
 
+		selectedFacetSites = Arrays.asList(ScopeFacetBuilder.getSelectedSites());
+
+		if (selectedFacetSites != null) {
+			for (Object item : selectedFacetSites) {
+				redirectURL = redirectURL + "&" + "site" + "=" + item;
```

Comment:
Hoping to have "site" replaced with a variable that will reference the facet name/type. It was mentioned that this ticket affects the tag facet as well.